scripts/populateInstances.js

```diff
@@ -142,7 +142,26 @@ const populateInstances = async ({
   // Write fetched webhooks to the disk
   fs.writeFileSync(outputWebhooksPath, JSON.stringify(webhooks, null, 2))
 
-  // Upload webhooks on the coordinating instance
+  // Create the supervisor instance
+  log('Destroying instance', supervisingInstanceDomain)
+  try {
+    await exec(
+      `cozy-stack instances destroy ${supervisingInstanceDomain} --force`
+    )
+  } catch (err) {
+    log('Instance does not exist')
+  }
+  await exec(
+    `cozy-stack instances add --apps drive,photos ${supervisingInstanceDomain} --passphrase cozy`
+  )
+  await exec(
+    `cozy-stack instances modify ${supervisingInstanceDomain} --onboarding-finished`
+  )
+  await exec(
+    `cozy-stack apps install --domain ${supervisingInstanceDomain} dissecozy file://${process.cwd()}/build/`
+  )
+
+  // Upload webhooks on the supervisor instance
```

Comment:
The querier is the supervisor, right? So now we might have 2 terms for the same thing?

Reply:
Querier = Node that performs the final aggregation, Supervisor = Node that receives observations from nodes participating in the execution of the protocol. The supervisor can be outside of the tree.

Querier might be abandoned with the dynamic tree creation: an array of node is passed and the created tree will have a single top level aggregator that corresponds to the querier in DISSEC-ML protocols.